Cargo.toml

```diff
@@ -44,3 +49,7 @@ switcheroo-control = { git = "https://github.com/pop-os/dbus-settings-bindings"
 zbus = { version = "4.2.1", default-features = false, features = ["tokio"] }
 unicode-truncate = "1.0.0"
 unicode-width = "0.1.11"
+
+[patch."https://github.com/pop-os/launcher/"]
+pop-launcher = { git = "https://github.com/wiiznokes/launcher/", rev = "86a54d54a68b832d404ef83a03af84cf9ef3e694" }
```

Comment:
I'm assuming we won't want to merge this reference to a fork into master.

If https://github.com/pop-os/launcher/pull/227 doesn't cause any regressions for the pop-shell frontend on 22.04, then I can regression test that one and we can get it merged first, if that would help?

Reply:
I will look into it. I must say I run into this problem, but the problem it's just the target/target (should be one target), so it's easy to work around it.

If you want to test the other pr, you must also install this one for it to work

Reply:
should be fixed. We probably want to merge
1. https://github.com/pop-os/cosmic-protocols/pull/35
2. https://github.com/pop-os/launcher/pull/227
3. this pr